### ISSUE TITLE
install.sh doesn't evaluate paths

### ISSUE THREAD

**Issue by DrGo** [koka-lang/koka#412]
```
~> curl -sSL https://github.com/koka-lang/koka/releases/download/v2.6.0/install.sh | sh -s -- --prefix=~/.local
Installing koka v2.6.0 for  macos-arm64
Installing dependencies..
Downloading: https://github.com/koka-lang/koka/releases/download/v2.6.0/koka-v2.6.0-macos-arm64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 48.4M  100 48.4M    0     0  4566k      0  0:00:10  0:00:10 --:--:-- 3210k
Unpacking..
Installing to prefix: ~/.local
- install executable            : ~/.local/bin/koka-v2.6.0
- install executable symlink    : ~/.local/bin/koka
- install pre-compiled libraries: ~/.local/lib/koka/v2.6.0
- install source libraries      : ~/.local/share/koka/v2.6.0
- install vscode editor support..
Install successful.

--------------------------------------------------
Installed Koka v2.6.0 at ~/.local/bin/koka

Type 'koka' to enter the interactive compiler

~> koka
zsh: command not found: koka
~> ~/.local/bin/koka
zsh: no such file or directory: /Users/[username]/.local/bin/koka
```
.local existed before, but none of the mentioned folders was created.
```
~/.local> uname -a
Darwin drlab 23.2.0 Darwin Kernel Version 23.2.0: Wed Nov 15 21:53:18 PST 2023; root:xnu-10002.61.3~2/RELEASE_ARM64_T6000 arm64
```

**Comment by TimWhiting** [comment]
I was able to reproduce: It looks like it ended up in /Users/[username]/~/.local/bin/koka. (As in a literal ~).

**Comment by daanx** [comment]
Thanks! I think this is fixed now (in `dev` but I updated the v2.6.0 release as well).
